assets/css/main.scss

```diff
@@ -54,3 +54,23 @@ p.sponsor-quote {
         max-width: 100%;
         box-shadow: 0 0 0 1px rgba(0,0,0,0.2);
 }
+
+a.site-title {
+    width: 30%;
+}
+
+.home .content {
+    margin-top: 2em;
+
+    > .logo {
+        width: 30%;
+        float: right;
+        display: inline-block;
+        margin-left: 2em;
+        margin-bottom: 2em;
+    }
+
+    div#sponsors {
```

Comment:
If this vertical whitespace looks good over the top of the Sponsors `<h3>`, you might want to consider applying it to every `<h3>` on the website, both for consistency and for the avoidance of having too many ad hoc rules that may making figuring out style things later harder.